build.sbt

```diff
@@ -10,24 +10,25 @@ ThisBuild / crossScalaVersions := Seq(Scala212, Scala3, Scala213)
 ThisBuild / tlVersionIntroduced := Map("3" -> "1.0.3")
 ThisBuild / tlCiReleaseBranches := Seq("main")
 ThisBuild / tlSiteApiUrl := Some(url("https://www.javadoc.io/doc/org.typelevel/mouse_2.13/latest"))
+ThisBuild / githubWorkflowOSes := Seq("ubuntu-22.04")
```

Comment:
Actually you can continue to use the default here.

http4s and FS2 needed `ubuntu-22.04` because they have additional requirements on a TLS library that I was having trouble using on ubuntu 20.04.

https://github.com/aws/s2n-tls

TLS and cryptography are the only requirement we cannot easily reimplement in pure Scala, that would be a bad idea 😅 so they are a little more hassle unfortunately.

Reply:
Hmm, it's good to know. Thanks!